updater/lib/dependabot/sentry.rb

```diff
@@ -1,29 +1,29 @@
 # frozen_string_literal: true
```

Comment:
Since this file is no longer using `RavenProcessor`,  I should actually further move out the sentry-specific bits to the caller and keep this as a generic exception message processing class.

That will also make testing it a bit easier since I can test solely against exception strings.

This also means the looping through multiple exceptions should get moved to the caller, but I already have to do that anyway because of the other processing we need to do in `before_send`, like applying `raven_context`.